Dockerfile

```diff
@@ -42,6 +44,8 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         cmake \
         curl \
         emacs25-nox \
+        esl-erlang \
```

Comment:
Could you put these in alphabetic order? Thanks!

Reply:
Doh. Done